packages/react-composites/src/composites/localization/locales/en-GB/strings.json

```diff
@@ -328,7 +328,7 @@
   },
   "chat": {
     "chatListHeader": "In this chat",
-    "uploadFile": "Upload File"
+    "uploadAttachment": "Upload File"

```

Comment:
Are we planning to adjust the localizable strings too?

Reply:
good point, i will update en-us, and left all other languages to TD pipeline to handle